Sources/CardPayments/Models/CardRequest.swift

```diff
@@ -11,14 +11,24 @@ public struct CardRequest {
     /// 3DS authentication launch option
     public let sca: SCA
     
+    /// The option to vault with purchase
+    public let shouldVault: Bool
+    
+    /// The customerID for vault with purchase option. If left blank, PayPal endpoint will auto-generate one
```

Comment:
The backing PayPal service will only generate a customerID if `shouldVault` is set to `true`, right? We should clarify this.

Reply:
I am not sure if it is internally generated upon capture/authorize without vault option, but vault field is not returned in capture/authorize result if shouldVault in request is false,